examples/api/src/index.ts

```diff
@@ -113,12 +114,34 @@ async function asyncMain() {
     }
   ]
 
+  if (!process.env.MAILGUN_API_KEY) throw new Error('No MAILGUN_API_KEY defined in environment.')
```

Comment:
Is this only for the example? and I am using another adaptor I will remove these validations?